daml-lf/spec/daml-lf-1.rst

```diff
@@ -149,10 +149,10 @@ Version: 1.7
     - add `NUMERIC` primitive type
     - add `numeric` primitive literal
     - add numeric builtins, namely `ADD_NUMERIC`, `SUB_NUMERIC`,
-      `MUL_NUMERIC`, `DIV_NUMERIC`, `ROUND_NUMERIC`, `CAST_NUMERIC`,
-      `SHIFT_NUMERIC`, `LEQ_NUMERIC`, `LESS_NUMERIC`, `GEQ_NUMERIC`,
-      `GREATER_NUMERIC`, `TEXT_TO_NUMERIC`, `NUMERIC_TO_TEXT`,
-      `INT64_TO_NUMERIC`, `NUMERIC_TO_INT64`, `EQUAL_NUMERIC`
+      `MUL_NUMERIC_LEGACY`, `DIV_NUMERIC_LEGACY`, `ROUND_NUMERIC`, `CAST_NUMERIC_LEGACY`,
```

Comment:
I would explain why we now have legacy functions, or alternatively, change the public specification later.

Reply:
In an upcoming PR